modules/conflict_resolver/test/conflict_resolverTest.php

```diff
@@ -34,9 +36,9 @@ function testConflictResolverDoespageLoad()
     {
         $this->webDriver->get($this->url . "/conflict_resolver/");
```

Comment:
I've noticed that a lot of the tests are using `safeGet` instead of `webDriver->get`. I haven't really looked into the differences, but thought I would point it out.